### PR TITLE
Add ActiveAdminImport::Exception

### PR DESCRIPTION
**Pull request by linqueta**
How was discussed in the issue #174 , this exception will let who consumes the gem create a custom error to threat it as a failure in import.

### CODE REVIEW

**Comment by linqueta** [comment]
Is happening this error for Rails version 5.2.2.1:

```ruby
Failure/Error: require 'active_admin'
NameError:
  uninitialized constant Responders::FlashResponder::ActionView
  Did you mean?  ActionDispatch
```

Do I have to do something?

**Comment by Fivell** [comment]
@linqueta hard to say, will take a look next week

**Comment by linqueta** [comment]
> @linqueta hard to say, will take a look next week

I'll look too on this weekend.

**Comment by linqueta** [comment]
> @linqueta hard to say, will take a look next week

Hey @Fivell , I fixed the problem in #176 , it was simple, just requiring 'action_view'. If you approve, I'll rebase it.

**Comment by Fivell** [comment]
Hei @linqueta please rebase and lets merge after it

File: spec/import_spec.rb [hunk 1]
```diff
@@ -430,6 +430,24 @@ def upload_file!(name, ext = 'csv')
...
+          expect(page).to have_content I18n.t('active_admin_import.no_file_error')
+          expect(Author.count).to eq(0)
+        end
+      end
+
+      context 'when the option before_batch_import raises a ActiveAdminImport::Exception' do
+        let(:options) { { before_batch_import: ->(_) { raise ActiveAdminImport::Exception, 'error message' } } }
+
+        it 'should show error' do
+          expect(page).to have_content I18n.t('active_admin_import.no_file_error')
```
Comment (Fivell) [inline spec/import_spec.rb]: @linqueta can u comment this
I18n.t('active_admin_import.no_file_error')
 is
 "Error: %{message}"

Comment (Fivell) [reply spec/import_spec.rb]: expect(page).to have_content I18n.t('active_admin_import.no_file_error', message: 'error message')
maybe this is valid expection

**Comment by Fivell** [comment]
@linqueta  good please squash commits and lets merge/release

**Pull request merged**